grpc/core/src/main/java/io/helidon/grpc/core/ResponseHelper.java

```diff
@@ -0,0 +1,453 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.grpc.core;
+
+import java.util.concurrent.Callable;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionException;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import io.grpc.stub.StreamObserver;
+
+/**
+ * A number of helper methods to handle sending responses to a {@link StreamObserver}.
+ */
+public interface ResponseHelper {
```

Comment:
This interface is a utility class.
Please reimplement as a utility class with static methods.
We do not want to add multiple inheritance into Helidon. Also this adds a lot of methods to the class (and as soon as reflection or CDI is used to obtain an instance, it makes the whole thing a lot more complicated).